packages/core/src/pubsub/pubsub-message.ts

```diff
@@ -34,7 +35,13 @@ export type ResponseMessage = {
   tips: Map<string, CID>
 }
 
-export type PubsubMessage = UpdateMessage | QueryMessage | ResponseMessage
+// All nodes will always ignore this message
```

Comment:
What do you mean by nodes will ignore this message? Should this not be tracked as a message with event type "received"?

Reply:
It should get logged the same as any pubsub message, but the ceramic node doesn't do any processing of it